client.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type StatusGoClient struct {
+	rpcClient *rpc.Client
+}
+
+func newClient(url string) (*StatusGoClient, error) {
+	rpcClient, err := rpc.Dial(url)
+	if err != nil {
+		return nil, err
+	}
+	return &StatusGoClient{rpcClient}, nil
+}
+
+func (c *StatusGoClient) getPeers() ([]Peer, error) {
+	peers := make([]Peer, 0)
```

Comment:
```suggestion
	var peers []Peer
```
to avoid allocating memory if the result is `nil`.

Reply:
k